okta/workflows-templates
# OKTA-467419: add generated screenshots of all the flows to each template

Resolves: https://oktainc.atlassian.net/browse/OKTA-467419

### Purpose

We need to generate previews of all of the flows in each template, so we can include them later in a new feature we're working on in the new catalog app.

### Testing:

- Make sure the screenshots inside each `template/resources` folder actually belong to that template
- Make sure that these screenshots are accurate and don't have any visual defeacts
